evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -81,7 +81,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * funds without the owner's intervention.)
    */
   uint256 constant private RESERVE_ROUNDS = 2;
-  uint256 constant private MAX_ORACLE_COUNT = 42;
+  uint256 constant private MAX_ORACLE_COUNT = 77;
```

Comment:
Wow we're really going for broke here ;)

Reply:
It was the most we could fit in under 400k.